elements/lisk-p2p/src/p2p.ts

```diff
@@ -12,7 +12,7 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-
+import { randomBytes } from 'crypto';
```

Comment:
Can we import this from `lisk-cryptography` as we have discussed here? https://github.com/LiskHQ/lisk-sdk/pull/4030#discussion_r309680239

Reply:
I also responded to your linked thread, but in short `getRandomBytes` is not exported to top level. Therefore here we can use `crypto` instead.

Reply:
lisk-cryptography is not exposing those functions.
Please expose and update the lisk-cryptography version to `2.3.0-alpha.0`